docker/compose.testing.yaml

```
@@ -0,0 +1,121 @@
+      - MINIO_ROOT_USER=${MINIO_ROOT_USER:?error}
+    # https://docs.docker.com/reference/dockerfile/#healthcheck
+    # https://docs.docker.com/reference/compose-file/services/#healthcheck
+    healthcheck:
+      disable: true
+    # https://docs.docker.com/reference/compose-file/services/#networks
+    networks:
+      - minio_test
+      - postgres_test
+      - redis_test
+    # https://docs.docker.com/reference/compose-file/services/#restart
+    restart: no
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)